src/test/java/org/fundacionjala/coding/kevincristhian/CesarCryptoTest.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * test.
+ */
+public class CesarCryptoTest {
+
+    private static final int NEGATIVE_THREE = -3;
+    private static final int NEGATIVE_FIVE = -5;
+    private static final int THREE = 3;
+    private static final int FIFTY_TWO = -52;
+    private static final int ZERO = 0;
+    /**
+     * encodeVigenere tests.
+     */
+    @Test
+    public void encodeVigenere() {
+        CesarCrypto cesarCrypto = new CesarCrypto();
+        Assert.assertEquals("TXYPXMUNHCRC", cesarCrypto.encodeVigenere("HOLAJALASOFT", "LIMON"));
+        Assert.assertEquals("TXYP XMUNHCRC", cesarCrypto.encodeVigenere("HOLA JALASOFT", "LIMON"));
+    }
+
+    /**
+     * decode encode cesar tests.
+     */
+    @Test
+    public void decodeCesar() {
+        CesarCrypto cesarCrypto = new CesarCrypto();
+        Assert.assertEquals("HOLA", cesarCrypto.decode("KROD", NEGATIVE_THREE));
+        Assert.assertEquals("ZORRO", cesarCrypto.decode("ETWWT", NEGATIVE_FIVE));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "ZORRO" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)